package.json

```diff
@@ -24,21 +24,16 @@
   },
   "scripts": {
     "compile": "tsc -p .",
+    "typecheck": "tsc -p . --noEmit",
     "build": "yarn compile && cp ./boilerplate/.gitignore ./boilerplate/.gitignore.template",
-    "format": "yarn format:js && yarn format:json && yarn format:md && yarn format:ts;",
-    "format:js": "prettier --write '**/*.js'",
-    "format:json": "prettier --write '**/*.json'",
-    "format:md": "prettier --write '**/*.md'",
-    "format:ts": "prettier --write '**/*.ts{,x}'",
-    "lint": "eslint src test --ext .ts --fix && yarn format",
+    "format": "prettier '**/*{.js,.ts,.tsx,.json,.md}'",
+    "format:write": "yarn format --write",
+    "format:check": "yarn format --check",
+    "lint": "eslint 'src/**' 'test/**'",
     "test": "jest",
     "watch": "jest --watch",
     "watch:debug": "yarn watch --runInBand --verbose",
     "coverage": "jest --coverage",
-    "ci:prep": "yarn build && yarn ci:lint",
-    "ci:lint": "eslint src test --ext .ts && yarn format",
-    "ci:test:vanilla": "jest ./test/vanilla",
-    "ci:test": "yarn ci:prep && yarn test && yarn clean",
```

Comment:
These scripts aren't used anymore. Most of the other scripts can be run for other purposes than CI